test/parallel/test-fs-readfilesync-enoent.js

```diff
@@ -1,13 +1,19 @@
 'use strict';
 const common = require('../common');
+
+// This test ensures fs.realpathSync works on properly on Windows without
+// throwing ENOENT when the path involves a fileserver
+// https://github.com/nodejs/node-v0.x-archive/issues/3542
+
 // This test is only relevant on Windows.
-if (!common.isWindows)
-  common.skip('Windows specific test.');
 
 const assert = require('assert');
 const fs = require('fs');
 const path = require('path');
 
+if (!common.isWindows)
+  common.skip('Windows specific test.');
```

Comment:
Nit: I don't think this should be moved. I think it should be left where it was.